DockDoor/Views/Hover Window/SharedPreviewWindowCoordinator.swift

```diff
@@ -64,7 +64,7 @@ final class SharedPreviewWindowCoordinator: NSWindow {
         isMovableByWindowBackground = false
         collectionBehavior = [.canJoinAllSpaces]
         backgroundColor = .clear
-        hasShadow = false
+        hasShadow = true
```

Comment:
This has been changed at https://github.com/ShlomoCode/DockDoor/commit/9c27ffa29f1e88bb57240b89943d0cd07178966e. Is this a deliberate change?

Reply:
Yes